client/src/app/ingredient.ts

```diff
@@ -0,0 +1,6 @@
+//Test list of data, will be replaced with DB
+
+export interface Ingredient {
```

Comment:
I think we should create a `FridgeService` at this point. Store the service and related files under `client/src/services/fridge`. @StevieShibly8 @NilinReza What do you guys think?

Reply:
FridgeService implemented for getting available ingredients and local storage ingredients